README.md

```diff
@@ -75,6 +75,17 @@ Then click "Add More Scanning Tools."
 
 ![image](/images/install04.png)
 
+### Workflow Setup - Organization Wide
+The scorecard workflow can be automatically added to every repository under an organization using the [multi-repo-action](https://github.com/ossf/scorecard-action/tree/main/multi-repo-action) tool.
```

Comment:
Suggested rewording (minor):

`To add the Scorecard workflow to every repository under an organization, use the multi-repo-action tool. Follow these steps: